e2e/fragments/addressPostcodeLookup.js

```diff
@@ -33,7 +33,10 @@ module.exports = {
   },
 
   enterAddressManually(address) {
-    I.click(this.cantEnterPostcodeLink);
+    if (this.cantEnterPostcodeLink.length > 0) {
```

Comment:
I can see some of the e2es are now failing around entering postcode, the only thing I can think would change is this. Maybe worth looking into? Easiest and quickest way to run e2e is to use an existing caseId in the correct state and hardcode it rather than creating a new case :)

Reply:
I spent some time trying to add a conditional into the applicant event page e2e yesterday, eventually discussed this with Adam and Tomeck and it's been decided that if reference data don't populate the address then we aren't going to populate it in the e2e. so resolving this. green build now.